.github/workflows/ci.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
               bash remrem_publish_event.bash -d $DOMAIN -n $NAMESPACE
 
               : # Wait for a while until the processing of the events
-              echo "Waiting..." && sleep 60
+              echo "Waiting..." && sleep 90
```

Comment:
Actually, we give some time to the Eiffel events process and the Jenkins jobs to be executed. A minute is not enough always, so I increased to 1.5 min.

Reply:
I don't question that we have to wait I was wondering if we could poll the Jenkins jobs or something that knows when the task is ready instead of just doing a sleep and hope that it will finish in time.

Reply:
We check if the Jenkins jobs are done in the next pipeline step "Ensure that Jenkins jobs executed successfully".